tools/SourceKit/lib/SwiftLang/SwiftEditor.cpp

```diff
@@ -1489,8 +1488,8 @@ class CodeFormatter {
           Builder.append(FmtOptions.IndentWidth, ' ');
       }
       Builder.append(Line);
-      Consumer.recordFormattedText(Builder.str().str());
-      return std::pair<SwiftEditorLineRange, StringRef> (SwiftEditorLineRange(LineIndex, 1), Text);
+      return std::pair<SwiftEditorLineRange, std::string> (SwiftEditorLineRange(LineIndex, 1),
```

Comment:
Remove space between > and (.  Also, this should be a call to `make_pair` probably.